packages/perseus-core/src/data-schema.ts

```diff
@@ -423,7 +423,7 @@ export type PerseusCategorizerWidgetOptions = {
     // Whether this widget is displayed with the results and immutable
     static: boolean;
     // The correct answers where index relates to the items and value relates to the category.  e.g. [0, 1, 0, 1, 2]
-    values: ReadonlyArray<number>;
+    values?: ReadonlyArray<number>;
```

Comment:
Potential concern here: with this change, the `data-schema` types no longer represent the JSON that actually gets persisted. The JSON should always have the `values` field, but the _client_ won't always receive it.

If we wanted to fix this, we would have to decouple the widget prop types from these data-schema types. Though I think maybe they are already pretty much decoupled (in theory) by the `transform` and `staticTransform` functions. `transform` renders the widget with public-only data and `staticTransform` renders with public+private data.

Reply:
I think if the JSON should always have the values field, then it might be worth it to fix this. It seems like the data-schema and our contract with users is pretty important, and I'd rather it be as accurate as possible. Aren't we creating different types for the JSON versus what the client is getting though?

Reply:
Yeah, I agree it's important for the data-schema to accurately reflect the JSON. You are right, the JSON types and the prop types are going to be different in a SSS world. I'll post in Slack with more thoughts.